py/sight/widgets/decision/current_status.py

```diff
@@ -30,13 +30,7 @@
 from sight_service.proto import service_pb2_grpc
 
 _LOG_ID = flags.DEFINE_string(
-    "log_id", None, "ID of the Sight log that tracks this execution.")
-_DEPLOYMENT_MODE = flags.DEFINE_enum(
```

Comment:
are we removing deployment mode flag or it's moved to any other file?

Reply:
That flag is in py/sight/service_utils.py and that file is now being transitively linked to this one, so the flag definitions were colliding